go-controller/pkg/node/gateway_udn_test.go

```diff
@@ -336,9 +337,10 @@ var _ = Describe("UserDefinedNetworkGateway", func() {
 		kubeMock.On("UpdateNodeStatus", cnode).Return(nil)
 		err = testNS.Do(func(ns.NetNS) error {
 			defer GinkgoRecover()
-			mpLink, err := udnGateway.addUDNManagementPort()
+			mpLink, _, err := udnGateway.addUDNManagementPort()
```

Comment:
shouldn't we add a test case for ipv6 and ensure LinkAddrExist?

Reply:
we do a few lines below:
```go
exists, err = util.LinkAddrExist(mpLink, ovntest.MustParseIPNet("ae70::2/112"))

```